clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -779,16 +778,17 @@ public void testOffsetCommitSyncFailedWithRetriableThrowsTimeoutWhenRetryTimeExp
             new TopicPartition("topic", 1),
             new OffsetAndMetadata(0));
 
-        // Send offset commit request that fails with retriable error.
-        long expirationTimeMs = time.milliseconds() + retryBackoffMs * 2;
-        CompletableFuture<Void> commitResult = commitRequestManager.commitSync(offsets, expirationTimeMs);
+        // Send offset commit request.
+        long deadlineMs = time.milliseconds() + retryBackoffMs * 2;
+        CompletableFuture<Void> commitResult = commitRequestManager.commitSync(offsets, deadlineMs);
+
+        // Make the first request fail due to a missing coordinator.
```

Comment:
Same here as above, update this code now that we don't preemptive prune expired requests.